Dockerfile

```diff
@@ -94,9 +95,12 @@ COPY supervisor.conf /etc/supervisor/conf.d/
 
 # Entrypoint
 ADD start /app/bin/
-RUN chmod a+rx /app/bin/start
+COPY crontab.txt /app/crontab.txt
+RUN chmod a+rx /app/bin/start \
+    && crontab -u weblate /app/crontab.txt
 
 ENV DJANGO_SETTINGS_MODULE weblate.settings
+ENV WEBLATE_OFFLOAD_INDEXING 1
```

Comment:
This should be rather completely removed and hardcoded in the settings.py instead.